renderer/src/features/mcp-servers/components/card-mcp-server/server-actions/items/edit-configuration-menu-item.tsx

```diff
@@ -10,11 +12,25 @@ interface EditConfigurationMenuItemProps {
 export function EditConfigurationMenuItem({
   serverName,
 }: EditConfigurationMenuItemProps) {
-  const [isRunWithCommandOpen, setIsRunWithCommandOpen] = useState(false)
+  const [serverDialogOpen, setServerDialogOpen] = useState<{
+    local: boolean
+    remote: boolean
+  }>({
+    local: false,
+    remote: false,
```

Comment:
We only handle local or remote mcps. Why not have a single toggle?

Reply:
it will be part of the next refactor, for now I didn't change too much avoiding really huge PR